src/panels/config/integrations/integration-panels/ozw/ozw-config-network.ts

```diff
@@ -46,203 +29,61 @@ class OZWConfigNetwork extends LitElement {
 
   @property() public ozw_instance = 0;
 
-  @internalProperty() private _network?: OZWInstance;
-
-  @internalProperty() private _statistics?: OZWNetworkStatistics;
-
-  @internalProperty() private _status = "unknown";
-
-  @internalProperty() private _icon = mdiCircle;
-
   public connectedCallback(): void {
     super.connectedCallback();
     if (this.ozw_instance <= 0) {
       navigate(this, "/config/ozw/dashboard", true);
     }
-    if (this.hass) {
-      this._fetchData();
-    }
   }
 
-  private async _fetchData() {
-    this._network = await fetchOZWNetworkStatus(this.hass!, this.ozw_instance);
-    this._statistics = await fetchOZWNetworkStatistics(
-      this.hass!,
-      this.ozw_instance
-    );
-    if (networkOnlineStatuses.includes(this._network.Status)) {
-      this._status = "online";
-      this._icon = mdiCheckCircle;
-    }
-    if (networkStartingStatuses.includes(this._network.Status)) {
-      this._status = "starting";
-    }
-    if (networkOfflineStatuses.includes(this._network.Status)) {
-      this._status = "offline";
-      this._icon = mdiCloseCircle;
-    }
-  }
+  protected render(): TemplateResult {
+    const route = computeTail(this.route);
+
+    const ozwTabs: PageNavigation[] = [
+      {
+        translationKey: "ui.panel.config.ozw.navigation.network",
+        path: `/config/ozw/network/${this.ozw_instance}/dashboard`,
+        iconPath: mdiServerNetwork,
+      },
+      {
+        translationKey: "ui.panel.config.ozw.navigation.nodes",
+        path: `/config/ozw/network/${this.ozw_instance}/nodes`,
+        iconPath: mdiNetwork,
+      },
+    ];
 
-  private _generateServiceButton(service: string) {
-    return html`
-      <ha-call-service-button
-        .hass=${this.hass}
-        domain="ozw"
-        service="${service}"
-      >
-        ${this.hass!.localize("ui.panel.config.ozw.services." + service)}
-      </ha-call-service-button>
-    `;
-  }
+    if (route.path !== "/nodes") {
```

Comment:
Why is this?

Reply:
If I include `<hass-tabs-subpage>` there then on the /nodes list `<hass-tabs-subpage-data-table>` I end up with 2 tab bars at the top

Reply:
Then move the `<hass-tabs-subpage>` to the elements underneath

Reply:
We should drop this component and only use `ozw-network-router`